server.py

```diff
@@ -268,14 +268,15 @@ def initialize_mysql(test=False):
             conn = engine.connect()
         else:
             conn = connect_mysql('root', '', '127.0.0.1', 3306, 'nyudevops')
-
+            
+debug = (os.getenv('DEBUG', 'False') == 'True')
+port = os.getenv('PORT', '5000')
```

Comment:
I don't think this is a good idea from a coding prospective to globalize these variables.
For the sake of coverage
using the coverage config files
https://stackoverflow.com/questions/5850268/how-to-test-or-mock-if-name-main-contents
should be better?